front-vite/src/views/EventListingView.vue

```diff
@@ -472,15 +474,17 @@ const currentRoute = useRoute()
 /**
  * Données des événements / animations
  */
+const itemsPerPage = 5
 const totalEvents = ref(0)
 const numberOfPages = ref(0)
+const currentPage = ref(0)
 const events = ref<{id: number}[]>([])
 const loading = ref(true)
 const options = ref({
   sortBy: ['begin_date'],
   sortDesc: [false],
   page: 0,
-  itemsPerPage: 10
+  itemsPerPage: itemsPerPage
```

Comment:
```suggestion
  itemsPerPage
```

Tu peux directement écrire cette notation abrégée, ça fait une "auto affectation" de la propriété `itemsPerPage` à la valeur de la variable `itemsPerPage` définie ligne 477.